src/alfasim_sdk/__init__.py

```diff
@@ -255,6 +255,7 @@ def get_alfasim_sdk_api_path():
 from alfasim_sdk._internal.constants import PipeThermalModelType
 from alfasim_sdk._internal.constants import PipeThermalPositionInput
 from alfasim_sdk._internal.constants import PumpType
+from alfasim_sdk._internal.constants import PVTCompositionalViscosityModel
```

Comment:
Do you have any idea of how those did sneak in in the wrong order?
Don't we run precommit on ci?

Reply:
Not sure, but I've sorted them in this PR